README.md

```diff
@@ -1,5 +1,12 @@
 # Variational-DL
-### Contributors 
+
+#### Variational Deep Learning is a method of deep learning where we use Neural Networks to generate data, instead of drawing conclusions from it.
+
+#### We have currrently implemented two Autoencoders:
```

Comment:
Use checkboxes to list out all the AEs which are going to be uploaded and tick them once uploaded.

Reply:
Refer to NLP repository

Reply:
Added the checkboxes